### ISSUE TITLE
Banner Icon URL support

### ISSUE THREAD

**Issue by DryIce1** [noatpad/obsidian-banners#53]
Hi @noatpad,

**Feature Request**
- add support for banner icon from a URL.

**Use cases**
1. The discord user Rainbell uses a CSS snippet and banners to insert an icon of the **current** date using a snippet. [It looks great on their homepage](https://github.com/Rainbell129/Obsidian-Homepage). The URL cand be modified and a YYYY-MM-DD date can be entered to give a static date.
    - It would be amazing to be able to use this in a daily note, so that the icon always reflects the note's date.
    - Here is the link to the calendar icon, and I have changed the URL to a static date 2022-04-19.
    - https://api.wolai.com/v1/icon?type=1&locale=en&date=2022-04-19&pro=0&color=blue
      <img src="https://api.wolai.com/v1/icon?type=1&locale=en&date=2022-04-19&pro=0&color=blue" width="100">
    - use templates/templater to insert the note's date.
2. Photos URLs as icons would be awesome regardless, and particularly useful for adding People Profile Pictures (related to #39), or Company Logos, or just giving a profile picture to a random note about, for example; 
    - The Earth: <img src="https://images.theconversation.com/files/379516/original/file-20210119-15-tmyvf4.jpg?ixlib=rb-1.1.0&q=45&auto=format&w=1200&h=1200.0&fit=crop" width="100"> 
    - the Structure of DNA: <img src="https://toppng.com/uploads/preview/dna-clipart-clipart-transparent-background-dna-11562900711cchqqhrzmy.png" width="100">
    - the Three Toed Sloth: <img src="https://i.natgeofe.com/n/e4adbc90-bbdd-4b0d-8de5-8cd2073ea870/three-toed-sloths_02_3x4.jpg" width="100">

Thanks for considering!

**Comment by noatpad** [comment]
Yeah this is something that I wanna do soon! URLs and local images are something I want the icon field to receive, and it shouldn't be too much of a trouble to do. It's related to #39 and is on my roadmap

**Comment by pennpeng** [comment]
+1
